.github/workflows/aws.yml

```
@@ -28,14 +28,14 @@ name: Deploy to Amazon ECS
 
 on:
   push:
-    branches: ["main","dev"]
+    branches: [ "main","dev" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)